invenio_communities/communities/services/service.py

```diff
@@ -663,17 +664,26 @@ def read(self, identity, id_, expand=False, include_deleted=False):
         """Retrieve a record."""
         record = self.record_cls.pid.resolve(id_)
         result = super().read(identity, id_, expand=expand)
-
+        is_user = False
+
+        if record.tombstone is not None:
+            owners = [
+                m.dumps() for m in Member.get_members(record.id) if m.role == "owner"
+            ]
+            is_user = any(
+                owner.get("user_id") == int(record.tombstone.removed_by.get("user"))
```

Comment:
I think that this is redundant as at the moment only admins can delete a community providing a tombstone!